py/selenium/webdriver/common/options.py

```diff
@@ -29,15 +29,16 @@ def __init__(self) -> None:
         super().__init__()
         self._caps = self.default_capabilities
         self._proxy = None
-        self.set_capability("pageLoadStrategy", "normal")
+        self.capabilities = ("pageLoadStrategy", "normal")
         self.mobile_options = None
 
     @property
     def capabilities(self):
         return self._caps
 
-    def set_capability(self, name, value) -> None:
```

Comment:
This is a breaking change, btw.

Reply:
Yes, the initial scope was to remove the duplicate methods in `wpwebkit`. After that I thought  converting `set_capability` to a setter method `capabilities` would be right choice. Since both the methods are used internally in `Options` class, I did not had to do changes in other places. If it's too risky change, you can ignore this change and probably merge the commit where only the duplicate attributes were removed.

Reply:
Well, it is not in the scope and breaks the API for current users. It also makes it confusing because `capabilities` is a very ambiguous name across the project and code base.

It'd be easier if you revert the changes and I am happy to merge what was considered in the initial scope.

Reply:
Sure.. No problem. I will revert the changes and re-commit.

Reply:
@diemol can you please check now..